packages/cli/src/lib/versioning/Lockfile.ts

```diff
@@ -120,7 +150,7 @@ export class Lockfile {
     };
 
     for (const [name, allEntries] of this.packages) {
-      if (filter && !filter(name)) {
+      if (typeof name !== 'string' || (filter && !filter(name))) {
```

Comment:
This is an outcome of the previous "__metadata" special key. Probably should be handed up there perhaps.

Reply:
now skipping if key initially if is one of the "special" yarn keys.